packages/attention/src/component.tsx

```diff
@@ -137,54 +141,58 @@ export function Attention(props: AttentionProps) {
   useAutoUpdatePosition(targetEl, isShowing, attentionEl, autoUpdatePosition, attentionState)
   
   return (
-    <div
-      data-testid='attention-el'
-      className={classNames(
-        {
-          [ccAttention.notCallout]: !props.callout,
-          invisible: !isVisible && !props.callout,
-          hidden: !isVisible && !props.tooltip,
-        },
-        className
-      )}
-      ref={attentionEl}
-    >
-      <div
-        role={props.role === '' ? undefined : props.tooltip ? 'tooltip' : 'img'}
-        aria-label={
-          ariaLabel === '' ? undefined : ariaLabel ?? defaultAriaLabel()
-        }
-        className={wrapperClasses}
-        id={props.id}
-      >
-        {!props.noArrow && (
-          <Arrow {...props} ref={arrowEl} direction={actualDirection} />
-        )}
-        <div className={ccAttention.content}>{props.children}</div>
-        {canClose && (
-          <button
-            type='button'
-            aria-label={i18n._(
-              /*i18n*/ {
-                id: 'attention.aria.close',
-                message: 'Close',
-                comment: 'Aria label for the close button in attention',
-              }
-            )}
-            onClick={onDismiss}
-            onKeyDown={(event) => {
-              if (!props.onDismiss) return
-              if (event.key === 'Escape') {
-                props.onDismiss()
-              }
-            }}
-            className={ccAttention.closeBtn}
+    <>
```

Comment:
I don't think we need this? It would at least make the diff much smaller :D

Reply:
I tried to remove `<><>` but I get this error then: `This expression is not callable. Type '{}' has no call signatures.ts. Property assignment expected.` 😕

Reply:
The curly braces (`{}`) can be removed from the return statement when you remove `<>`, and that should fix it :)

Reply:
Updated in this commit: [7039695](https://github.com/warp-ds/react/pull/233/commits/7039695d139f54d1f290b4157e5ee0842a166128)